server.js

```diff
@@ -88,6 +88,7 @@ app.get('/conference', function(req, res) {
   res.render('desktop/conference', {
     authenticated: utils.isAuthenticated(req),
     editions: [
+      { year: '2017', path: 'http://reflectionsprojections.org/' },
```

Comment:
@traugertyson Can you change the path here to 'https://www-s.acm.illinois.edu/conference/2017' ?